tests/ssl-opt.sh

```diff
@@ -11516,6 +11516,102 @@ run_test    "TLS 1.3 m->G server with middlebox compat support, not client" \
             1 \
             -c "ChangeCipherSpec invalid in TLS 1.3 without compatibility mode"
 
+requires_openssl_tls1_3
```

Comment:
I would be great to add some negative tests:
1. one test where the Mbed TLS server is not able to find a suitable signature algorithm for the CertificateVerify message
2. the same on client side
3. one test where the Mbed TLS server is not able to find a suitable certificate given the list of signature algorithms provided by the client (ssl_tls13_pick_key_cert() fails).

Reply:
Oops, forget that. I have added some debug message at the point, but did not add test cases :(

Reply:
Please check.